source/renderer/app/components/loading/syncing-connecting/SyncingProgress/SyncingProgress.tsx

```diff
@@ -79,7 +79,7 @@ const SyncingProgress: FC<Props> = (props, { intl }: Context) => (
           key={type}
           className={makePercentageCellStyles(props[type] === 100)}
         >
-          {Math.floor(props[type])}%
+          {props[type]}%
```

Comment:
I moved this to the main process so we can compare against actually displayed values (previously we were getting values as float and then dropping the fractional part, which would results in unnecessary re-renders)